spacy/tests/lang/am/test_noun_chunks.py

```diff
@@ -0,0 +1,16 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import pytest
+
+
+def test_noun_chunks_is_parsed_es(es_tokenizer):
+    """Test that noun_chunks raises Value Error for 'am' language if Doc is not parsed.
+    To check this test, we're constructing a Doc
+    with a new Vocab here and forcing is_parsed to 'False'
+    to make sure the noun chunks don't run.
+    """
+    doc = es_tokenizer("በበጋ ወቅት በኦክስፎርድ")
```

Comment:
Typo: this test should use `am_tokenizer` (and the name of the test should be changed as well)

Reply:
☹️ after change UT fails. Looking into this further.